.gitlab-ci.yml

```
@@ -82,7 +94,7 @@ test-linux-stable:                 &test
 check-web-wasm:
   stage:                           test
   image:                           tomaka/cargo-web:latest
-  allow_failure: true
+  allow_failure:                   true
```

```suggestion
  allow_failure:                   false
```

i think it's fine to strictly require <= 120 characters now